pkg/controller/account/byoc.go

```diff
@@ -59,6 +56,61 @@ func claimBYOCAccount(r *ReconcileAccount, reqLogger logr.Logger, currentAcctIns
 	return nil
 }
 
+// Checks whether or not the access keys need to be rotated based on state, and if so, rotates them
+func (r *ReconcileAccount) initializeCredentials(reqLogger logr.Logger, awsSetupClient, client awsclient.Client, currentAcctInstance *awsv1alpha1.Account, accountClaim *awsv1alpha1.AccountClaim, adminAccessArn string) (string, error) {
+
+	err := r.byocRotateAccessKeys(reqLogger, client, accountClaim)
```

Comment:
This was removed in master, lets remove here as well

Reply:
Since this is being removed, does it make sense to remove this function totally?

Reply:
Removed `initializeCredentials` function and call to `byocRotateAccessKeys`.  Moved `createBYOCAdminAccessRole` to parent loop.

Removed `byocRotateAccessKeys` function as it is no longer used.